src/buildHTML.js

```diff
@@ -1598,15 +1598,17 @@ groupTypes.xArrow = function(group, options) {
 
     const arrowBody = stretchy.svgSpan(group, options);
 
-    const arrowShift = -options.fontMetrics().axisHeight + arrowBody.depth;
-    const upperShift = -options.fontMetrics().axisHeight - arrowBody.height -
-        0.111;    // 2 mu. Ref: amsmath.dtx: #7\if0#2\else\mkern#2mu\fi
+    const arrowShift = -options.fontMetrics().axisHeight +
+        0.5 * arrowBody.height;
+    // 2 mu kern. Ref: amsmath.dtx: #7\if0#2\else\mkern#2mu\fi
+    const upperShift = -options.fontMetrics().axisHeight -
+        0.5 * arrowBody.height - 0.111;
```

Comment:
Why the change in the shifts?

Reply:
In previous versions, I treated extensible arrows slightly differently than the other SVGs. If an extensible arrow had a total height of 0.522 ems, then the values I returned to `buildHTML` had a `arrowBody.height` of 0.261 and a `arrowBody.depth` of 0.261. Now the code in `stretchy.js` is so array-driven, I found it easier to make things more consistent. Now, even extensible arrows return `.height` with their overall height value and return `.depth` as zero.

So the code inside `buildHTML` had to adjust and split that value in half, so that the arrow is centered on the math axis.

Reply:
> Now the code in stretchy.js is so array-driven, I found it easier to make things more consistent. Now, even extensible arrows return .height with their overall height value and return .depth as zero.

Could you add a comment describing this in stretchy.js?

Reply:
Sure. I'll leave a note in both stretchy.js and in in buildHTML.js.